app/konnect/runtime-manager/runtime-groups/index.md

```diff
@@ -124,7 +124,7 @@ You can sort the table by the `Last Ping` column.
 Column | Description
 -------|-------------
 Host | The hostname of the instance.
-Type | The runtime instance type. Currently, only Kong Gateway is supported.
-Last Ping | The last time that this instance received a configuration update from the Konnect control plane. was used to proxy a Service.
-Version | The Kong Gateway version that this instance is running.
+Type | The runtime instance type. Currently, only {{site.base_gateway}} is supported.
+Last Ping | The last time that this instance received a configuration update from the {{site.konnect_short_name}} control plane. was used to proxy a Service.
```

Comment:
```suggestion
Last Ping | The last time that this instance received a configuration update from the {{site.konnect_short_name}} control plane to proxy a service.
```
There was a partial sentence at the end, so I took my best guess as to what it was supposed to be.